Models/Courier.cs

```diff
@@ -9,7 +11,71 @@ public class Courier
         public string? Username { get; set; }
         public string Password { get; set; } = string.Empty;
         public int CreatedBy { get; set; }
-        public string PhoneNumber { get; set; } = string.Empty; 
+        public string PhoneNumber { get; set; } = string.Empty;
         public Transport Transportation { get; set; }
+
+        public static int CheckIfValid(Courier courier)
+        {
+            if (courier.id == 0 || courier.Name == null || courier.Surname == null || courier.Email == null || courier.Username == null || courier.Password == null || courier.CreatedBy == 0 || courier.PhoneNumber == null || courier.Transportation == null)
```

Comment:
The id should not matter and not even be asked for. The server should create a unique id itself.

Reply:
You can use annotation for validation: 
https://learn.microsoft.com/en-us/aspnet/web-api/overview/formats-and-model-binding/model-validation-in-aspnet-web-api
And create custom model validation:
https://learn.microsoft.com/en-us/aspnet/core/mvc/models/validation?source=recommendations&view=aspnetcore-7.0#ivalidatableobject

Reply:
@Martynexs that will be useful to know for me in the future. Thanks for the links! For now ill leave it be how it is due to lack of time to change up everything.